### ISSUE TITLE
New template for location entry form

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1035]
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

[Geographic features](https://github.com/publiclab/plots2/milestone/7) are too numerous and complex for folks to make much progress on right now. Let's start with a self-contained module -- the HTML template for selecting a location. This is described and illustrated in this GIF from #650 -- so let's start there:

![gif](https://cloud.githubusercontent.com/assets/161439/17665075/8f4b041a-62c6-11e6-9fdf-0bf5b4a71e80.gif)

Before we can make any of this template in HTML, we need a place to put it in our Rails application. Let's:

1. create a new template people can visit and try out in a new file at `/app/views/locations/form.html.erb` which (I think?) should be accessible via https://publiclab.org/locations/form
2. if the above isn't accessible, then if needed, add an empty controller method in `/app/controllers/tag_controller.rb` (for now) called `def location` and a `config/routes.rb` entry to direct requests to that action appropriately: `  match 'locations/form' => 'tag#location'`
3. begin constructing the form in `form.html.erb` based on above gif, and on the form input examples in https://getbootstrap.com/css/#forms

For step 3, we can break out a new issue -- but just start with a simple template like:

```html
<h2>Enter a location</h2>
<p>This page does not yet work</p>

<form>
  <input type="text" class="form-control" placeholder="Begin typing a location" />
</form>
```

Later, we can create new issues for fleshing out that template more, and perhaps for making a new `/app/controllers/location_controller.rb` file instead of re-using the tag controller.

**Comment by manish310794** [comment]
@jywarren I would like to tackle this one.

**Comment by jywarren** [comment]
Great, please do! thank you!

**Comment by jywarren** [comment]
Just a note -- this will be part of a series of small issues broken out of this big list: https://github.com/publiclab/plots2/issues/1070

**Comment by ebarry** [comment]
Hi @manish310794,  just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by mridulnagpal** [comment]
Hi @manish310794 , if you are not working with this issue I would like to give it a go

**Comment by jywarren** [comment]
Hi, all -- I think it's fair for @mridulnagpal to take a crack at this -- @manish310794, I'm happy to help you find a new issue to tackle if you're still interested, and as this progresses towards a pull request, I'm sure there are also plenty of opportunities for collaboration. Thanks!

**Comment by mridulnagpal** [comment]
@jywarren I checked the files and the template already exists, please tell me what do I do next?

**Comment by jywarren** [comment]
Ah - i see it here: https://github.com/publiclab/plots2/blob/master/app/views/locations/form.html.erb

It was closed in #1044-- but we didn't link back or close this -- oops! We should remind people to do so in their PRs. 

Yes, the next step is the next item in #1070. I'll break out a new issue for it, thanks!

**Comment by jywarren** [comment]
Great - moved next step to #1107
